rusk-wallet/src/lib/store.rs

```diff
@@ -50,15 +56,29 @@ impl LocalStore {
         }
 
         // attempt to load and decode wallet
-        let bytes = fs::read(&path)?;
-        let bytes = decrypt(&bytes, pwd)?;
+        let mut version = 0u8;
+        let mut bytes = fs::read(&path)?;
 
-        if bytes.len() != SEED_SIZE {
-            return Err(Error::WalletFileCorrupted);
+        // check for version information (strips it if present)
+        if bytes[1] == 0 && bytes[2] == 0 {
```

Comment:
What happen if the 2nd and 3rd bytes are 0 but it's not a wallet?
Usually this is resolved using a MagicNumber at the beginning of the file

Reply:
This is a good point, but nothing will really happen, the user will just get a "File corrupted" error as expected.